README.md

```diff
@@ -355,6 +356,56 @@ parameter. This means that depending on its value, only the associated data (raw
+
+The default data model is the collection per entity one, but sometimes to get the best performance out of the MongoDB
+database where the raw and aggregated data is stored, alternative data models are needed. This is the reason why we
+introduced the data model migration tools.
+
+To set the desired data model to be used, please take a look at [Running the STH server](#section5) below.
+
+To run the data migration tool, please execute the following command:
+```
+./bin/sth_database_model
+```
+which will present the command help information:
+```
+Usage: sth_database_model [options]
+
+  Options:
+
+    -h, --help                         output usage information
+    -V, --version                      output the version number
+    -a, --analysis                     prints the results of the data model analysis including the databases and collections which need to be migrated to the currently configured data model (mandatory if not -m or --migrate)
+    -m, --migrate                      migrates to the currently configured data model all the databases and collections which has been created using a distinct data model (mandatory if not -a or --analysis)
+    -r, --remove-collection            the original data model collection will be removed to avoid conflict if migrating back to that data model in the future
+    -u, --update-collection            the migration will take place even if the target collections already exist combining the data of the original and target collections (use this option with special care since the migration operation is not idempotent for the aggregated data collections)
+    -d, --database <databaseName>      only this database will be taken into consideration for the analysis and/or migration process
+    -c, --collection <collectionName>  only this collection will be taken info consideration, a database is mandatory if a collection is set
+```
+
+Special care should be taken when requesting a data model migration since the migration of aggregated data is not an idempotent operation if the target data model collections already exist. In this case, the already existent data stored in these collections is combined with the one stored in the original data model collections pending migration. Based on this fact, we suggest the next procedure when considering a data model migration:
+
+1. Check the current configured data model used by the STH component based on the options detailed in [Running the STH server](#section5) below. Typically it will be the default collection per entity data model.
+2. Get a data model analysis report about the databases and collections which need to be migrated to the new desired data model running the following command: `LOGOPS_FORMAT=dev DATA_MODEL=collection-per-service-path ./bin/sth_database_model -a`. Use the desired data model instead of the one included in the previous example.
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-sth-comet/pull/312/commits/61fb711d0bfd7dc2855678322a6b1a8882a2fb67